packages/x-tailwindcss/src/x-tailwind-plugin/components/tag/tight.ts

```diff
@@ -0,0 +1,39 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the `tight` styles for component `tag`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+export function tagTight(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    tight: {
+      borderColor: 'transparent',
+      color: `var(--tag-color-75, ${theme('colors.neutral.90')})`,
+      paddingInlineStart: theme('spacing.0'),
+      paddingInlineEnd: theme('spacing.0'),
+
```

Comment:
Should be added `fontWeight: theme('fontWeight.regular')` even if its already added in the default one?

Reply:
I don't think so, I didn't added because is not a property that changes from the default styles.